mcf-browser/README.md

```diff
@@ -0,0 +1,83 @@
+
+If there is no node to be displayed, the the App component renders the Home component. If no files have been loaded to the application, the the Home component will soley display the FileEntry componet to prompt the use to upload files. Once files have been uploaded, the Home component displays lists of the loaded files and the subject nodes of each triple within the file(s). The Home component will also display the ParsingErrorsTable component which displays any errors that were encountered in parsing the files. The Home component also has a button which allows the user to reveal a collapsable FileEntry component if they would like to add more files to their KG.
+
+Brief description of each JSX file:
+  - App.jsx - Maintains the overall state of the application, including loaded files and which node should be displayed. Renders the Header component followed by DisplayNode if a node has been selected by the user and Home otherwise.
+  - Header.jsx - Displays the header containing the 'Return Home' button and search bar.
+  - Home.jsx - If no files have been indicated by user, displays file entry options. Otherwise, displays the currently loaded files and their subject nodes.
+  - DisplayNode.jsx - Retrieves data for a given dcid from the back-end and displays the triples associated with the node via TriplesTable components.
+  - TriplesTable.jsx - Renders a table where each row represents a triple from a given list of Assertions.
+  - FileEntry.jsx - Renders the options for how a user can upload files or specify the URL(s) for them.
+  - ParsingErrorsTable.jsx - Renders the errors found in parsing into a table format.
+  - LoadingSpinner.jsx - Simple component with spinning animation to indicate when a page is loading.
+  - utils.js - Contains helper functions to set the window's hash and various parameters in the url of the application.
+
+### Running Locally + Testing
+
+To run and test locally, you will need node.js and npm installed. If you do not have these, visit [node.js website](https://nodejs.org/en/download/) to download and install node.js, then install npm.
+
+You can run the following commands to ensure you have both node.js and npm installed:
+```sh
+$ node -v
+$ npm -v
+```
+
+After forking this repo, ensure you are in `/tools/mcf-browser`,
+then run
+```sh
+$ npm install
+```
+which will install the packages specified in `package.json`.
+
+After installing node_modules, you can run the tests for the back-end by running
+```sh
+$ npm run test
+```
+
+You can deploy a local version of the application to test the front-end with
+```sh
+$ npm start
+```
```

Comment:
Please add a note about deploying a new version as well (even if it's with github-pages -- any notes for us to deploy this in the future?)

Reply:
Note that we are now deploying the entire repo to github pages...
https://datacommonsorg.github.io/tools/imputation/

Could you add a note about this, and that we should try to move this elsewhere?